packages/client/src/__tests__/benchmarks/query-performance/seed-data.ts

```diff
@@ -0,0 +1,854 @@
+/*
+ * Data seeding utilities for query performance benchmarks.
+ * Creates realistic test data for various benchmark scenarios.
+ */
```

Comment:
A summary comment of how the seed test is structured would go a long way here.
I see we are basically creating a graph of relationships between models, and there's a given probability that one user profile follows another, or that a user has on average `n` posts. Let's clarify precisely the modelled scenario via a comment, thanks.